src/coinbase/tests/e2e.ts

```diff
@@ -0,0 +1,112 @@
+import fs from "fs";
+import dotenv from "dotenv";
+import { Coinbase } from "../coinbase";
+import { TransferStatus } from "../types";
+
+describe("Coinbase SDK E2E Test", () => {
+  let coinbase: Coinbase;
+  beforeAll(() => {
+    dotenv.config();
+  });
+
+  beforeEach(() => {
+    coinbase = new Coinbase({
+      apiKeyName: process.env.NAME,
+      privateKey: process.env.PRIVATEKEY,
+    });
+  });
+
+  it("should be able to access environment variables", () => {
+    expect(process.env.NAME).toBeDefined();
+    expect(process.env.PRIVATEKEY).toBeDefined();
+  });
+
+  it("should have created a dist folder for NPM", () => {
+    expect(fs.existsSync("./dist")).toBe(true);
+    expect(fs.existsSync("./dist/index.js")).toBe(true);
+    expect(fs.existsSync("./dist/client/index.js")).toBe(true);
+    expect(fs.existsSync("./dist/coinbase/coinbase.js")).toBe(true);
+  });
+
+  it("should be able to interact with the Coinbase SDK", async () => {
+    console.log("Fetching default user...");
+    const user = await coinbase.getDefaultUser();
+    expect(user.getId()).toBeDefined();
+    console.log(`Fetched default user with ID: ${user.getId()}`);
+
+    console.log("Creating new wallet...");
+    const wallet = await user.createWallet();
+    expect(wallet?.getId()).toBeDefined();
+    console.log(
+      `Created new wallet with ID: ${wallet.getId()}, default address: ${wallet.getDefaultAddress()}`,
+    );
+
+    console.log("Importing wallet with balance...");
+    const seedFile = JSON.parse(process.env.SEED || "");
```

Comment:
this should be process.env.SEED_FILE instead of process.env.SEED. It is confusing

Reply:
@jazz-cb sure, it is content of the seed file. Maybe SEED_FILE_CONTENT?